docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md

```diff
@@ -134,29 +134,27 @@ It does not support absolute time ranges: `2021-12-02 00:00:00 to 2021-12-05 23:
 
    To add multiple queries, click **Add query**.
 
-   All alert rules are managed by Grafana by default. If you want to switch to a data source-managed alert rule, click **Switch to data source-managed alert rule**.
-
 2. Add one or more [expressions].
 
-   a. For each expression, select either **Classic condition** to create a single alert rule, or choose from the **Math**, **Reduce**, and **Resample** options to generate separate alert for each series.
+   a. For each expression, select either **Classic condition** to create a single recording rule, or choose from the **Math**, **Reduce**, and **Resample** options.
 
    {{% admonition type="note" %}}
    When using Prometheus, you can use an instant vector and built-in functions, so you don't need to add additional expressions.
    {{% /admonition %}}
 
    b. Click **Preview** to verify that the expression is successful.
 
-3. To add a recovery threshold, turn the **Custom recovery threshold** toggle on and fill in a value for when your alert rule should stop firing.
+3. To add a recovery threshold, turn the **Custom recovery threshold** toggle on and fill in a value for when your recording rule should stop metting the condition.
 
-   You can only add one recovery threshold in a query and it must be the alert condition.
+   You can only add one recovery threshold in a query and it must be the recording rule condition.
 
-4. Click **Set as alert condition** on the query or expression you want to set as your alert condition.
+4. Click **Set as recording rule condition** on the query or expression you want to set as your rule condition.
 
 #### Set evaluation behavior
 
-Use alert rule evaluation to determine how frequently an alert rule should be evaluated and how quickly it should change its state.
+Use recording rule evaluation to determine how frequently an recording rule should be evaluated and how quickly it should change its state.
```

Comment:
```suggestion
Use recording rule evaluation to determine how frequently a recording rule should be evaluated and how quickly it should change its state.
```

Reply:
hey sonia-just a couple of minor things...plus added the backport label since it's been available from 11.3